examples/thermostat/thermostat-common/thermostat.zap

```diff
@@ -4293,7 +4309,7 @@
               "code": 0,
```

Comment:
And is ResetCounts() for Ethernet Network Diagnostics implemented in these apps?